cmd/dp-dataset-exporter/main.go

```diff
@@ -93,6 +69,15 @@ func main() {
 	eventConsumer := event.NewConsumer()
 	eventConsumer.Consume(kafkaConsumer, eventHandler, errorHandler)
 
+	neoHealthChecker := neo4j.NewHealthCheckClient(neo4jConnPool)
+	filterAPIHealthChecker := filterHealthCheck.New(config.FilterAPIURL)
```

Comment:
could just in-line these in the below function call?

Reply:
Done